### PR TITLE
Bug fixes: social share buttons

### LINKED ISSUE #434: Web Sharing Bugs
**Describe the bug**

- The WhatsApp and Facebook buttons should redirect to a new tab or open up a new smaller window like `react-share` does
- Copying link isn't working

### PR DESCRIPTION
**Pull request by anthonycruzmacedo**
## Summary

Fix functional and style bugs for social share buttons

Closes #434 

## Changes

* FB and WA buttons now open new windows
* Copy to clipboard bug fixed, working now
* Hide social share buttons on mobile
* Update screen size dependent stylings in projectDetailsStyles

## Screenshots

image

image

### CODE REVIEW

File: zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        >
+          <WhatsAppIcon />
+        </IconButton>
+        <IconButton
+          className={classes.button}
+          onClick={() => {
+            navigator.clipboard
+              .writeText(url)
+              .then(() => {
+                alert('Link copied to clipboard: ' + url);
```
Comment (NdibeRaymond) [inline zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx]: plain string should be localized. Also we use toaster, not native alert. you can refer to other places in the codebase where toaster is used.

File: zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+              })
+              .catch(() => {
+                alert('Something went wrong!');
```
Comment (NdibeRaymond) [inline zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx]: same here. localization and no use of native alert function

File: zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx [hunk 3]
```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { ButtonGroup } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import LinkIcon from '@material-ui/icons/Link';
+import WhatsAppIcon from '@material-ui/icons/WhatsApp';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import IconButton from '@material-ui/core/IconButton';
+import { useMediaQuery } from '@material-ui/core';
+
+const styles = makeStyles({
```
Comment (NdibeRaymond) [inline zubhub_frontend/zubhub/src/components/social_share_buttons/socialShareButtons.jsx]: style should be moved to the styles folder, just like is the case for other components

**Review by NdibeRaymond — CHANGES_REQUESTED** [review]

**Comment by anthonycruzmacedo** [comment]
@NdibeRaymond Hi Raymond, I have added localization features to the component and moved the styles to the right place.

**Review by ghost — APPROVED** [review]

**Pull request merged**